### PR TITLE
Add media attribute support for meta[name=theme-color]

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR adds upcoming "media" attribute support for `meta[name=theme-color]` to the "Add a web app manifest" article.
See https://groups.google.com/a/chromium.org/g/blink-dev

Live preview: https://deploy-preview-5620--web-dev-staging.netlify.app/add-manifest/#theme-color

### CODE REVIEW

File: src/site/content/en/progressive-web-apps/add-manifest/index.md [hunk 1]
```diff
@@ -254,6 +254,17 @@ the app's preview in task switchers. The `theme_color` should match the
...
 </figure>
 
+In Chrome 93, you can adjust this color based on a media query with the `media`
+attribute of the `meta` theme color element. As an example, you could have one
+color for light mode and another one for dark mode. At the time of writing, you
+can't define those in your manifest. See [w3c/manifest#975 GitHub
+issue](https://github.com/w3c/manifest/issues/975).
+
+```html
+<meta name="theme-color" media="(prefers-color-scheme: light)" content="white">
```
Comment (yoavweiss) [inline src/site/content/en/progressive-web-apps/add-manifest/index.md:264]: Seems like this should mention the ordering significance

Comment (beaufortfrancois) [reply src/site/content/en/progressive-web-apps/add-manifest/index.md:264]: Done.

File: src/site/content/en/progressive-web-apps/add-manifest/index.md [hunk 2]
```diff
@@ -254,6 +254,17 @@ the app's preview in task switchers. The `theme_color` should match the
   {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/8mkBdT3O0FZLo0PUppvv.png", alt="An example of a PWA window with custom theme_color.", width="800", height="196", class="w-screenshot" %}
 </figure>
 
+In Chrome 93, you can adjust this color based on a media query with the `media`
+attribute of the `meta` theme color element. The first one that matches will be
+picked. As an example, you could have one color for light mode and another one
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/add-manifest/index.md:259]:
```suggestion
picked. For example, you could have one color for light mode and another one
```

Comment (jpmedley) [reply src/site/content/en/progressive-web-apps/add-manifest/index.md:259]: This is more idiomatic.

File: src/site/content/en/progressive-web-apps/add-manifest/index.md [hunk 3]
```diff
@@ -254,6 +254,17 @@ the app's preview in task switchers. The `theme_color` should match the
   {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/8mkBdT3O0FZLo0PUppvv.png", alt="An example of a PWA window with custom theme_color.", width="800", height="196", class="w-screenshot" %}
 </figure>
 
+In Chrome 93, you can adjust this color based on a media query with the `media`
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/add-manifest/index.md:257]:
```suggestion
In Chrome&nbsp;93, you can adjust this color based on a media query with the `media`
```

Comment (jpmedley) [reply src/site/content/en/progressive-web-apps/add-manifest/index.md:257]: This is something Tom has been doing for a long time.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/progressive-web-apps/add-manifest/index.md (hunk 2 above)
Comment (beaufortfrancois) [reply src/site/content/en/progressive-web-apps/add-manifest/index.md:259]: Fixed.

File: src/site/content/en/progressive-web-apps/add-manifest/index.md (hunk 3 above)
Comment (beaufortfrancois) [reply src/site/content/en/progressive-web-apps/add-manifest/index.md:257]: Fixed.

**Review by jpmedley — APPROVED** [review]

**Comment by beaufortfrancois** [comment]
Thank you! I'll take care of merging when it is available in Chrome 93. See https://twitter.com/quicksave2k/status/1404447732287279107 for context

**Comment by beaufortfrancois** [comment]
As suggested by @tomayac, I've added the same paragraph to https://deploy-preview-5620--web-dev-staging.netlify.app/prefers-color-scheme/#reacting-on-dark-mode-changes

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 4]
```diff
@@ -465,6 +465,18 @@ in order to see the theme color and favicon changes, open the
   });
 ```
 
+In Chrome&nbsp;93, you can adjust the color based on a media query with the
```
Comment (tomayac) [inline src/site/content/en/blog/prefers-color-scheme/index.md:468]:
```suggestion
As of Chromium&nbsp;93 and Safari&nbsp;15, you can adjust the color based on a media query with the
```

Comment (tomayac) [reply src/site/content/en/blog/prefers-color-scheme/index.md:468]: (Same below of course.)

Comment (beaufortfrancois) [reply src/site/content/en/blog/prefers-color-scheme/index.md:468]: Done

**Review by tomayac — APPROVED** [review]
LGTM, approval

**Pull request merged**